lib/lacaml_utils.ml

```diff
@@ -147,6 +149,14 @@ external ilaenv : int -> string -> string -> int -> int -> int -> int -> int
 let check_var_ltz loc var_name var =
   if var < 0 then invalid_arg (sprintf "%s: %s < 0: %d" loc var_name var)
 
+let check_var_within loc var_name var lb ub c =
```

Comment:
Also not certain if this function belongs here, but I figured since I use it twice to check variables, it seemed natural.